src/Core/Header.php

```diff
@@ -192,6 +192,8 @@ private static function parseConfigFile($map, $selectedAssets = array())
                 if (($k == "bootstrap") && ((!in_array("no_main-theme", $selectedAssets)) || (in_array("patientportal-style", $selectedAssets)))) {
                     // Above comparison is to skip bootstrap theme loading when using a main theme or using the patient portal theme
                     //  since bootstrap theme is already including in main themes and portal theme via SASS.
+                } else if ($k == "compact-theme" && !$GLOBALS['enable_compact_mode']) {
```

Comment:
don't we need to also skip this if `in_array("no_main-theme", $selectedAssets)`

Reply:
Looks like a keystroke mistake on my part woops